### DISCUSSION TITLE
Syntax highlighting in vs code?

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [windicss/windicss#348] [Q&A] [answered]
Is there a plugin for vs code for windicss directives syntax highlighting?

**Comment by shiftgeist** [comment, marked as answer]
In what language do you want it to highlight?

CSS windicss highlighting eg `@apply` works for me.

**Reply by MadadiSaeed** [reply]
Which version are you using?

**Reply by shiftgeist** [reply]
3.1.3 with vscode 1.57.0

I don't think syntax highlighting is handled by windicss directly

**Reply by MadadiSaeed** [reply]
No. I mean vscode's version?

**Reply by shiftgeist** [reply]
vscode 1.57.0

**Reply by MadadiSaeed** [reply]
Thanks for your response. I did not know that windicss has an extenstion for vscode.
